x/auth/types/test_utils.go

```diff
@@ -44,6 +47,56 @@ func NewTestTx(ctx sdk.Context, msgs []sdk.Msg, privs []crypto.PrivKey, accNums
 	return tx
 }
 
+// TODO Rename to NewTestTx
+// TODO Is this impl better than using TxFactory?
```

Comment:
It seems like this is doing more or less the same exact logic as `tx.Factory.Sign()` right? I think it would make sense to create a helper function that is shared and could be used by tests too.